ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -59,27 +59,26 @@
                 var TBServiceCodeGroupState = TBServiceCodeError ? Error : Standard;
                 var TBServiceCodeSelectErrorState = TBServiceCodeError ? SelectType.Error : SelectType.Standard;
             }
+            @if (Model.Notification.NotificationStatus == NotificationStatus.Draft) 
+            {
             <nhs-form-group ref="formGroupTBServiceCode" nhs-form-group-type=@TBServiceCodeGroupState>

```

Comment:
Indenting looks a bit off - did you try formatting the page? (Ctrl + Shift + F in VSCode I believe)

Reply:
Sadly I don't have a formatter installed for vscode atm for aspnetcorerazor files, I'll look into getting one! Hopefully this is fixed now anyway